core/src/test/java/com/datastax/oss/driver/internal/core/control/ControlConnectionTest.java

```diff
@@ -525,7 +532,7 @@ public void should_close_channel_if_closed_during_reconnection() {
     channel1.close();
     waitForPendingAdminTasks();
     Mockito.verify(eventBus).fire(ChannelEvent.channelClosed(node1));
-    Mockito.verify(reconnectionSchedule).nextDelay();
+    Mockito.verify(reconnectionSchedule).nextDelay(Optional.empty());
```

Comment:
Maybe we can name that `Optional.empty()` to state the intent of the test? 
like `static Optional<T> NO_DELAY`